src/components/Summary/DocumentsDelivery.vue

```diff
@@ -86,7 +86,7 @@ export default class DocumentsDelivery extends Mixins(CommonMixin) {
 
   private validateEmailFormat (value: string): boolean {
     // allow empty as the email is optional
-    if (value === '') {
+    if ((value === '') || (value === undefined)) {
```

Comment:
What if value is `null` ?

https://developer.mozilla.org/en-US/docs/Glossary/Falsy